src/www/firewall_rules_edit.php

```diff
@@ -1202,6 +1209,50 @@ function is_posnumericint($arg) {
                         </div>
                       </td>
                   </tr>
+<?
+$priorities = [
```

Comment:
we use array(), not []

If you create this array, map the values as well:

'' => gettext("None"),
'1' => gettext("Background"),
etc.